api/sheets/get_name.md

```diff
@@ -6,41 +6,60 @@ parent: Google Sheets
 grand_parent: API
 ---
 
-# Get a node's name
+# Returning the Names Associated with a DCID
 
-Given a cell or a column range of cells with [DCIDs](/glossary.html#dcid) (unique identifiers for all entities in the Data Commons knowledge graph), return the names associated with the given DCIDs.
+The `=DCGETNAME(*dcids*)` formula returns the names associated with given DCIDs to a cell or a column range of cells.
 
-## General information about this formula
+> **NOTE:**
+> 
+> Be sure to follow the instructions for [Installing and Enabling the Sheets Add-On](https://docs.datacommons.org/api/sheets/) before using this formula.
 
-**Formula**: `=DCGETNAME(dcids)`
+## Formula {#formula}
 
-**Required arguments**:
+```
+=DCGETNAME(*dcids*)
+```
 
-* `dcids` - A list of nodes to query, identified by their DCIDs.
+## Required Arguments {#required-arguments}
 
-## Examples
+`dcids` - A list of DCID nodes to query
 
-Before trying this method out, make sure to follow the setup directions in [the main section for Sheets docs](/api/sheets/index.html).
+## Returns {#returns}
 
-### Example 1: Retrieve the name of a country by its DCID.
+The formula returns the name associated with a specified DCID. See the [Examples](#examples) section below for examples of positive returns. If a DCID does not exist, the `=DCGETNAME(*dcids*)` formula does not return a value. For example, because the “geoId/123123123” DCID does not exist, no value is returned to cell B1 in the following sheet:
```

Comment:
```suggestion
The formula returns the name associated with a specified DCID. See the [Examples](#examples) section below for examples of positive return values.

If a DCID does not exist, the `=DCGETNAME(*dcids*)` formula does not return a value. For example, because the “geoId/123123123” DCID does not exist, no value is returned to cell B1 in the following sheet:
```